node/config/def.go

```diff
@@ -98,6 +98,10 @@ func DefaultBoost() *Boost {
 			},
 
 			MaxTransferDuration: Duration(24 * 3600 * time.Second),
+
+			HttpTransferMaxConcurrentDownloads: 20,
+			HttpTransferStallTimeout:           Duration(30 * time.Second),
```

Comment:
I'd set this much higher, maybe 5min, then tune based on what we observe. 30s is likely to happen quite often for many random reasons (temporary link drops, bgp reshufflings, temporary lock/io contention on either end)